qcut/resources/default-skills/native-cli/references/reference-vimax.md

```
@@ -13,7 +13,7 @@ All `vimax:*` commands share these override flags:
 | `--video-model` | Override video generation model |
 | `--output-dir`, `-o` | Output directory |
 
-## Default LLM Model
+### Default LLM Model
```

markdownlint-cli2 (0.21.0)

[warning] Heading levels should only increment by one level at a time
Expected: h2; Actual: h3

(MD001, heading-increment)